<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据管理</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QPushButton" name="manageBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chartBtn">
      <property name="text">
       <string>销售统计</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="passwordBtn">
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>59</y>
     <width>700</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="managePage">
    <widget class="QListWidget" name="dailyList">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>301</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>商品管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>321</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>317</width>
        <height>289</height>
       </rect>
      </property>
      <attribute name="label">
       <string>出售商品</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>10</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类  型</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sellOkBtn">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>257</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sellCancelBtn">
       <property name="geometry">
        <rect>
         <x>197</x>
         <y>257</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>58</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>品  牌</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>104</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>单  价</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>153</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数  量</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>200</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金  额</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellSumLineEdit">
       <property name="geometry">
        <rect>
         <x>103</x>
         <y>200</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="sellNumSpinBox">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>150</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="sellTypeComboBox">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>8</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="sellBrandComboBox">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>56</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>187</x>
         <y>102</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLabel" name="sellLastNumLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>152</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余数量：000</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>256</x>
         <y>202</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellPriceLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>101</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>317</width>
        <height>289</height>
       </rect>
      </property>
      <attribute name="label">
       <string>已有商品入库</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>317</width>
        <height>289</height>
       </rect>
      </property>
      <attribute name="label">
       <string>新商品入库</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="chartPage">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>销售统计</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="typeComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="uptBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>更新显示</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>manageBtn</tabstop>
  <tabstop>chartBtn</tabstop>
  <tabstop>passwordBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
